src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotSuitableSDOCallbackHandlerTest.java

```diff
@@ -60,42 +61,28 @@ public class NotSuitableSDOCallbackHandlerTest extends BaseCallbackHandlerTest {
     class AboutToStartCallback {
 
         private CallbackParams params;
-        private CaseData caseData;
-        private String userId;
 
         private static final String EMAIL = "example@email.com";
-        private LocalDateTime startedDate;
 
         @BeforeEach
         void setup() {
-            caseData = CaseDataBuilder.builder().atStateClaimDraft().build();
+            CaseData caseData = CaseDataBuilder.builder().atStateClaimDraft().build();
             params = callbackParamsOf(caseData, ABOUT_TO_START);
-            userId = UUID.randomUUID().toString();
+            String userId = UUID.randomUUID().toString();
 
             given(idamClient.getUserDetails(any()))
                 .willReturn(UserDetails.builder().email(EMAIL).id(userId).build());
 
-            startedDate = LocalDateTime.now();
-            startedDate = LocalDateTime.of(startedDate.getYear(), startedDate.getMonth(), startedDate.getDayOfMonth(),
-                             startedDate.getHour(), startedDate.getMinute(), startedDate.getSecond(),
-                             0);  // set to avoid elision of zeroes to cause random test errors.
-
-            given(time.now()).willReturn(startedDate);
+            given(time.now()).willReturn(LocalDateTime.now());
 
         }
 
-        @SuppressWarnings("unchecked")
         @Test
         void checkUnsuitableSDODate() {
```

Comment:
This is the test that was causing problems. With the JacksonConfiguration class now this test can be simplified.